packages/path_provider/path_provider_foundation/lib/path_provider_foundation.dart

```diff
@@ -68,4 +75,21 @@ class PathProviderFoundation extends PathProviderPlatform {
   Future<String?> getDownloadsPath() {
     return _pathProvider.getDirectoryPath(DirectoryType.downloads);
   }
+
+  /// Returns the path to the container of the specified App Group.
+  /// This is only supported for iOS.
```

Comment:
Nit: There should be a blank line above this per https://dart.dev/guides/language/effective-dart/documentation#do-separate-the-first-sentence-of-a-doc-comment-into-its-own-paragraph